src/main/resources/interface-specification.yml

```diff
@@ -75,6 +75,12 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+        '429':
```

Comment:
I wonder which is the use case of 420 "enhance your calm" if 429 is the right status code here